test/sharness/t0180-pubsub.sh

```diff
@@ -57,6 +57,36 @@ test_expect_success "wait until echo > wait executed" '
   test_cmp expected actual
 '
 
+test_expect_success "wait for another pubsub message" '
+  echo "testOK2" > expected &&
+  mkfifo wait2 ||
+  test_fsh echo init fail
+
+  # ipfs pubsub sub is long-running so we need to start it in the background and
+  # wait put its output somewhere where we can access it
+  (
+    ipfsi 2 pubsub sub --enc=ndpayload testTopic | if read line; then
+        echo $line > actual &&
+        echo > wait2
+      fi
+  ) &
+'
+
+test_expect_success "wait until ipfs pubsub sub is ready to do work" '
+  go-sleep 500ms
```

Comment:
Should we poll here for something (to be sure that pubsub is ready) or not?

Reply:
I see that we do it in other places so LGTM either way.

Reply:
Ideally, but thats pretty difficult to do. For now, this is good enough